pypeit/spectrographs/keck_mosfire.py

```diff
@@ -72,10 +72,8 @@ def default_pypeit_par(self):
         par['calibrations']['wavelengths']['fwhm']= 5.0
         par['calibrations']['wavelengths']['n_final']= 4
         par['calibrations']['wavelengths']['lamps'] = ['OH_NIRES']
-        #par['calibrations']['wavelengths']['nonlinear_counts'] = self.detector[0]['nonlinear'] * self.detector[0]['saturation']
         par['calibrations']['wavelengths']['method'] = 'holy-grail'
```

Comment:
Have you tested holy grail on other setups besides Y and J? It would be good to do so.